docs/ios-xcuitest.md

```diff
@@ -0,0 +1,13 @@
+## The XCUITest Driver for iOS
```

Comment:
this server args section should probably just go in the README somewhere. it was titled this way because it used to be in the general Appium docs. Now it is in the XCUITest driver so we don't need to tell people that it is.